README.md

```diff
@@ -530,14 +530,28 @@ Uses [behave](https://github.com/behave/behave) to execute BDD scenarios that ar
 
 Builds the `sdist` and `wheel` archives of this project using `poetry build`. It also generates a `requirements.txt` file which is useful when installing the package in a Docker container where you may want to install the dependencies in a specific Docker layer to optimize caching.
 
+##### install #####
+Publishes the `pom.xml` for the module into your local Maven Repository ('~/.m2.repository`).
```

Comment:
A (Administrative): bad tick syntax

Reply:
Any reasoning for the `~/.m2.repository` instead of `~/.m2/repository`. Also this `'~/.m2.repository` current begins with a ' instead of `.

Reply:
Nope - good catch.  Fixed!